src/Command/Integration/IntegrationCommandBase.php

```diff
@@ -455,48 +462,41 @@ private function getFields()
                 'description' => 'The Splunk event source type',
                 'required' => false,
             ]),
+            'protocol' => new OptionsField('Protocol', [
+                'conditions' => ['type' => ['syslog']],
+                'description' => 'Syslog transport protocol',
+                'required' => false,
+                'default' => 'tls',
+                'options' => ['tcp', 'udp', 'tls'],
+            ]),
             'host' => new Field('Host', [
                 'optionName' => 'syslog-host',
                 // N.B. syslog is an actual PHP function name so this is wrapped in extra array brackets, to avoid is_callable() passing
                 'conditions' => ['type' => ['syslog']],
                 'description' => 'Syslog relay/collector host',
-                'default' => 'localhost',
-                'required' => false,
                 'autoCompleterValues' => ['localhost'],
             ]),
             'port' => new Field('Port', [
                 'optionName' => 'syslog-port',
                 'conditions' => ['type' => ['syslog']],
                 'description' => 'Syslog relay/collector port',
```

Comment:
```suggestion
                'description' => 'Syslog relay/collector port',
                'autoCompleterValues' => ['6514'],
```

this could probably be autocompleted as it's the default for TLS in many places?